### ISSUE TITLE
[feature] appear online to specific contacts

### ISSUE THREAD

**Issue by ammarabdulsalam** [emesene/emesene#533]
Hello,
In windows live messenger 2011 there is a new feature "appear online to specific contacts".
It's working like block all, but still can receive offline messages from contacts you're appearing offline to them.
I think it would be nice to add this feature :-)

**Comment by c10ud** [comment]
i have no idea on how's this implemented, but i guess it depends on MSNP21 while papyon is MSNP18, so it's a no go

**Comment by ghost** [comment]
What you would need to do is essentially block all other contacts except the one you wanted to appear online to. I'm not sure, but I think this is similar how WLM 2011 did this feature as using it always messed with my block list.

**Comment by Sbte** [comment]
It'd probably be nice to have the block list editor like in emesene 1 and then make it possible to select multiple contacts at the same time. That's essentially the same feature, but then better.
